src/advanced/bitcoin-anchoring.md

```diff
@@ -214,13 +215,22 @@ secured.
 
 #### Synchronization with Bitcoin Blockchain
 
-A separate [method][handler] performs regular synchronization of the list of
-Exonum anchoring transactions with those committed to the Bitcoin Blockchain. It
-takes the latest
-anchoring transaction and checks whether it is present in Bitcoin. If not, the
-handler checks the previous anchoring transactions one by one in the same manner
-until it finds the last anchoring transaction committed to Bitcoin Blockchain.
-The handler then pushes all the uncommitted anchoring transactions to Bitcoin.
+A separate [btc_anchoring_sync][btc_anchoring_sync] utility periodically performs
```

Comment:
Nit: `btc_anchoring_sync` should be code-formatted, although I'd remove the name altogether and linked from "A separate utility" instead.